app/src/app/components/contents/runner/run/PacketRunForm.tsx

```diff
@@ -41,7 +41,7 @@ export const PacketRunForm = ({ defaultBranch, branches, mutate }: PacketRunForm
       parameters: []
     }
   });
-  const selectedBranch = branches.find((branch) => branch.name === form.getValues("branch"));
+  const selectedBranch = branches.find((branch) => branch.name === form.watch("branch"));
```

Comment:
fix issue where switching between branches in select didn't actually change branch

Reply:
updated test as requested